model/patch/cli_intent.go

```diff
@@ -21,7 +21,7 @@ type cliIntent struct {
 	DocumentID string `bson:"_id"`
 
 	// PatchFileID is the object id of the patch file created in gridfs
-	PatchFileID mgobson.ObjectId `bson:"patch_file_id"`
+	PatchFileID mgobson.ObjectId `bson:"patch_file_id,omitempty"`
```

Comment:
My understanding is that it will only make a difference on the db side. With or without `omitempty` when we unmarshal the bson back into an intent struct we will get an empty ObjectId.

Reply:
yes and no, the main danger would be queries looking for patch_file_id: "" which would need to switch to exists: false

Reply:
ah, you're right.
I don't see any queries involving the patch_file_id key.